GoogleForCreators/web-stories-wp
# Updated alignment buttons, icons and tooltips

## Context

This updates the layout, buttons, icons, and tooltips in the alignment panel.

## User-facing changes

| Before | After |
|-|-|
| ![Screen Shot 2021-03-01 at 23 18 20](https://user-images.githubusercontent.com/637548/109596653-abe55e00-7ae4-11eb-9356-9e029f822fee.png) | ![Screen Shot 2021-03-02 at 11 23 10](https://user-images.githubusercontent.com/637548/109679629-bf76e000-7b49-11eb-86fb-6f58dfdfcb44.png) |

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Select 3 or more elements on a page
2. Observe all the buttons in the alignment panel function and look as desired

### UAT




## Checklist




---

Fixes #6467
